web/backend/src/Server.ts

```diff
@@ -182,23 +183,30 @@ app.post('/api/logout', (req, res) => {
   });
 });
 
+// This function helps us convert the double list of the authorization
+// returned by the casbin function getFilteredPolicy to a map that link
+// an object to the action authorized
+// list[0] contains the policies so list[i][0] is the sciper
+// list[i][1] is the object and list[i][2] is the action
+function setMapAuthorization(list: string[][]) {
+  const m = new Map<String, Array<String>>();
```

Comment:
do you want to use `String` the object or just `string` the plain data type?

Reply:
When searching which one use, I found that it is better to avoid bugs to use `string`